setup.py

```diff
@@ -1,4 +1,7 @@
 import setuptools
+import subprocess
+
+subprocess.run(["python3", "pre-publish.py"])
```

Comment:
Since we not using PyPI for storing prebuild artifacts we need to call it before building from the source.